packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -51,6 +52,12 @@ class TestRenderingFlutterBinding extends BindingBase with Scheduler, Services,
     if (phase == EnginePhase.paint)
       return;
     renderView.compositeFrame();
+    if (phase == EnginePhase.composite)
```

Comment:
Ah, ok.

LGTM.

I expected it to literally be the same enum, as opposed to two identical enums, but this is fine.